h2olog

```diff
@@ -710,6 +710,9 @@ def handle_quic_event(cpu, data, size):
     if allowed_quic_event and ev.type != allowed_quic_event:
         return
 
+    if allowed_res_header_name and ev.type == "send_response_header" and ev.h2o_header_name != allowed_res_header_name:
```

Comment:
Is this condition what you really intended? My take is that this condition would allow HTTP-level `send_response_header` events to be printed even when `-s foo` is omitted. Would something like this make more sense?

```
if ev.type == "send_response_header":
    if allowed_res_header_name == None or ev.h2o_header_name != allowed_res_header_name:
        return
```

Reply:
One more thing, if we adopt the above condition, we'll no longer need the verbose option (at least for this round).

Reply:
Oh, it has broken by https://github.com/toru/h2olog/pull/18/commits/13146a8e3b2c3f6a907e83c7541e05c5676439da and fixed in 320974b23e382f99b0b2dfbed22340e7686120fa

My intention is:

* `-v` enables HTTP-level events (it *might* include `receive_request_header` in the future)
* If `-v` is specified and`-s` is omitted, all the HTTP-level events are printed
* if `-v` and `-s` are specified, only specified response headers

I think `-v` should exist if `h2olog quic` handles only `quic` tracing by default, but it can be removed if we follow https://github.com/toru/h2olog/issues/21.

Reply:
I see your point. Thank you for the fix, and walking me through your reasoning. This makes sense to me. I also appreciate your flexibility:

> but it can be removed if we do #21.

Agreed. There's no need to be fixated on that option if we do #21.